modern-gpus/amd-gpu.md

```diff
@@ -202,16 +222,16 @@ Extras:
 
 ## Unsupported AMD GPUs
 
-### **Navi 20 series**
+### **Navi 2X**
 #### Highest Supported OS: None
 
-These GPUs are slated for support sometime in the next few macOS releases as we have full confirmation on framebuffer and PCI ID support in the drivers. However, at this moment, there is limited functionality with these GPUs so we recommend holding off purchasing until proper acceleration has been achieved in the public.
+While Navi 21 is supported, the other cores are not at the time of writing.
 
 Unsupported Cards:
 
-* RX 6800
-* RX 6800 XT
-* RX 6900 XT
+* RX 6700 XT
+* RX 6600 XT
+* RX 6600
```

Comment:
RX 6600 non-XT is confirmed working under 12.1: https://www.tonymacx86.com/threads/rx-6600-xt-support.315477/page-2#post-2297131